include/ppx/geometry.h

```diff
@@ -152,47 +152,37 @@ class Geometry
 
         BufferType  GetType() const { return mType; }
         uint32_t    GetElementSize() const { return mElementSize; }
-        uint32_t    GetSize() const { return CountU32(mData); }
-        void        SetSize(uint32_t size) { mData.resize(size); }
```

Comment:
Any reason to remove `SetSize`? This would break Use # 1 in the comment above this class.

Reply:
The only reason was this is not used anywhere, so untested, so I removed it.
I can either remove the comment, or restore this.

Reply:
Angela will need this, so re-adding it.